mod/file/actions/file/download.php

```diff
@@ -6,6 +6,6 @@
  */
 
 // @todo this is here for backwards compatibility (first version of embed plugin?)
-$download_page_handler = elgg_get_plugins_path() . 'file/download.php';
+$download_page_handler = dirname(dirname(__DIR__)) . '/download.php';
```

Comment:
I'd like to see us register routes for this instead of putting files at magic locations on the filesystem... but that's a separate issue.